config/galaxy.ini.sample

```diff
@@ -1113,6 +1113,13 @@ use_interactive = True
 # Details" option in the history.  Administrators can always see this.
 #expose_dataset_path = False
 
+# This option allows users to see the job metrics (except for environment
+# variables).
+#expose_job_metrics = False
+
+# This option allows users to see the built command line.
+#expose_job_command_line = False
```

Comment:
I feel like we should reuse ``expose_dataset_path`` for ``expose_job_command_line`` and we could rename ``expose_job_metrics`` to ``expose_potentially_sensitive_job_metrics`` so we could one day default to exposing the ``core`` metrics for instance - I feel like this one should just be displayed to everyone regardless of setting up extra things.